src/index.html

```diff
@@ -1,13 +1,368 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
+    <link
+          rel="icon"
+          type="image/x-icon"
+          href="./images/favicon.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;700&family=Poppins:wght@400;500;700&display=swap"
+          rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+
+  <body class="page__body">
+    <header class="header" id="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+
+            <a href="#home" class="logo">
+              <img
+              class="logo__image"
+              src="../src/images/logo.svg"
+              alt="Take the Streets"
+              >
```

Comment:
Bad if you can’t fix your code style 🥲

```suggestion
              <img
                class="logo__image"
                src="../src/images/logo.svg"
                alt="Take the Streets"
              >
```